recipes/sos-notebook/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f3ae4371f2a9cf493c77190010e73270ddc658904618937889ce14c0ecebb09
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
+
+requirements:
+  host:
+    - ipython
+    - notebook >=5.0.0
+    - pip
+    - python >=3.6
+  run:
+    - ipykernel
+    - ipython
+    - markdown
+    - matplotlib
+    - nbconvert >=5.1.1
+    - nbformat
+    - notebook >=5.0.0
+    - pandas
+    - python >=3.6
+    - setuptools
+    - sos >=0.19.2
```

Comment:
OK, fiddling this for a long time... because `sos` is not in the default channel, and conda build [does not yet support specifying channels for requirements](https://stackoverflow.com/questions/25610843/conda-build-requirement-add-package-from-specific-channel), I will have to run

```
conda config --add channels conda-forge 
```
before running
```
conda build sos-notebook
```

It is a shame that conda-forge does not allow dependency from conda-forge.

Reply:
> It is a shame that conda-forge does not allow dependency from conda-forge.

Huh? This is not true

Reply:
As I described, `conda-forge` packages are allowed, but users have to run

```
conda config --add channels conda-forge
```
outside of `conda build`, rather than allowing things like

```
channels:
   - conda-forge
```
or
```
requirements:
    - conda-forge:
        - sos
```
in the configuration file.

I apologize for my "shame" word but it was frustrating to spend a few hours debugging this simple problem.

Reply:
At least the error message could be more informative such as "package xxx cannot be found in channels xxxx".